src/commands/Highlight/import.ts

```diff
@@ -0,0 +1,106 @@
+import { Command, CommandOptions, KlasaMessage, KlasaGuild } from 'klasa';
+import { ApplyOptions } from '@skyra/decorators';
+import { MessageEmbed, Util } from 'discord.js';
+import { pluralize } from '../../lib/utils/Util';
+
+@ApplyOptions<CommandOptions>({
+	aliases: ['i'],
+	description: 'Import your words from a different Guild.',
+	permissionLevel: 2,
+	runIn: ['text'],
+	usage: '<regex|words:default> <Guild:guild>',
+	extendedHelp: [
+		"→ If you want to import words from a guild",
+		'`{prefix}import [words] 479665229396049920` → Specifying `words` is optional as it is the default subcommand',
+		"",
+		"→ Importing your regular expressions from a guild",
+		"`{prefix}import regex 479665229396049920` → Adds your regular expressions from that guild, if they aren't added already.",
+	].join('\n'),
+})
+export default class extends Command {
+	needsMember = true;
+
+	async words(message: KlasaMessage, [guild]: KlasaGuild[]) {
+		if (!message.guild || !message.member) throw new Error('Unreachable');
+
+		const member = await guild.members.fetch(message.author);
+
+		if (!member) {
+			return message.send(new MessageEmbed()
+				.setColor(0x43B581)
+				.setDescription('You do not appear to be in that guild.'),
+			);
+		}
+
+		const words = member.settings.get('words') as string[];
```

Comment:
Consider adding an await member.settings.sync(true) here and on regexes for ensuring we are fetched